diff --git a/node_modules/@remix-run/dev/dist/vite/cloudflare-proxy-plugin.d.ts b/node_modules/@remix-run/dev/dist/vite/cloudflare-proxy-plugin.d.ts
index a99ecf9..d691224 100644
--- a/node_modules/@remix-run/dev/dist/vite/cloudflare-proxy-plugin.d.ts
+++ b/node_modules/@remix-run/dev/dist/vite/cloudflare-proxy-plugin.d.ts
@@ -1,5 +1,7 @@
 import { type AppLoadContext } from "@remix-run/server-runtime";
+// @ts-ignore
 import { type Plugin } from "vite";
+// @ts-ignore
 import { type GetPlatformProxyOptions, type PlatformProxy } from "wrangler";
 type CfProperties = Record<string, unknown>;
 type LoadContext<Env, Cf extends CfProperties> = {
